native/cocos/editor-support/MiddlewareManager.cpp

```diff
@@ -80,24 +80,24 @@ void MiddlewareManager::update(float dt) {
         auto *editor = _updateList[i];
         editor->update(dt);
     }
+}
 
-    for (auto &iter: _operateCacheMap) {
+void MiddlewareManager::render(float dt) {
+    // _deferredDestroy(called from ts) may be triggered after update, and native object may be release at this time.
```

Comment:
`Object._deferredDestroy` is called after component update in `Director.tick` and before emitting `BEFORE_DRAW` event in which `MiddlewareManager::render` is invoked, so the native object may be released here and it needs to be erased from `_updateList`.